lib/dependabot/file_updaters/java_script/npm_and_yarn/yarn_lockfile_updater.rb

```diff
@@ -347,6 +342,17 @@ def handle_missing_package(package_name)
             raise PrivateSourceAuthenticationFailure, reg
           end
 
+          def raise_resolvability_error(error, yarn_lock)
+            # This happens if a new version has been published that relies on
+            # subdependencies that have not yet been published.
+            raise error if resolvable_before_update?(yarn_lock)
```

Comment:
Pretty sure this will always return false, will always run the yarn updater with the new versions.
Could we just get rid of this check? Do we not want to also raise `DependencyFileNotResolvable` in the above case?

Reply:
Good point - updated.